src/configs/database.config.js

```diff
@@ -1,31 +1,29 @@
-'use strict';
-
 const mongoose = require('mongoose');
 const logger = require('~logger');
 
 const { DB_CONN, DB_NAME } = process.env;
 
-const config = (opts = {}) => {
-	const uri = getConnectionUrl(false);
-	mongoose
-		.connect(
-			uri,
-			Object.assign({ useNewUrlParser: true, useUnifiedTopology: true }, opts)
-		)
-		.then(() => logger.info('Connection to database successful!'))
-		.catch(err => logger.error(`Database error: ${err.message}`));
+const getConnectionUrl = (forTest = false) => {
+  const host = `${DB_CONN || 'mongodb://localhost:27017'}`.replace(/\/$/, '');
+  const dbName = DB_NAME || 'traceph';
+  if (forTest) {
+    return `${host}/test-${dbName}`;
+  }
+  return `${host}/${dbName}`;
 };
 
-const getConnectionUrl = (forTest = false) => {
-	const host = `${DB_CONN || 'mongodb://localhost:27017'}`.replace(/\/$/, '');
-	const dbName = DB_NAME || 'traceph';
-	if(forTest) {
-		return `${host}/test-${dbName}`
-	}
-	return `${host}/${dbName}`;
-}
+const config = (opts = {}) => {
+  const uri = getConnectionUrl(false);
+  mongoose
+    .connect(
+      uri,
+      { ...opts, useNewUrlParser: true, useUnifiedTopology: true },
```

Comment:
Make the unwrapping last so we can override the defined options if we want to.
```suggestion
      { useNewUrlParser: true, useUnifiedTopology: true, ...opts },
```